backend/services/esweService.ts

```diff
@@ -391,4 +392,42 @@ export default class EsweService {
     }
     return createFlaggedContent(null)
   }
+
+  async getWorkHistoryDetails(nomisId: string): Promise<workHistoryFullDetails> {
+    if (!app.esweEnabled) {
+      return createFlaggedContent(null)
+    }
+
+    try {
+      const context = await this.systemOauthClient.getClientCredentialsTokens()
+      const oneYearAgo = moment().subtract(1, 'year').format('YYYY-MM-DD')
```

Comment:
Not a blocker, but the code to get the work history to show shouldn't be duplicated code.

This code and the code above have the same reason for change - if this changes the other one must. Therefore they should be in one place.